.github/workflows/unit_test.yml

```diff
@@ -30,6 +30,24 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -r requirements.txt
+      - name: Check out repository
+        uses: actions/checkout@v4
+      - name: Set environment variables for MATLAB Engine
+	if: ${{ runner.os == 'Linux' || runner.os == 'macOS' }}
+        run: |
+          if [[ "${{ runner.os }}" == "macOS" ]]; then
```

Comment:
I don't see this variable set in the examples. What is it for? And does windows not need it?